examples/bsc-example/src/app/bsc.tx.example.ts

```diff
@@ -1,6 +1,7 @@
 import { TatumBscSDK } from '@tatumio/bsc'
+import { REPLACE_ME_WITH_TATUM_API_KEY } from '@tatumio/shared-testing-common'
 
-const bscSDK = TatumBscSDK({ apiKey: '75ea3138-d0a1-47df-932e-acb3ee807dab' })
+const bscSDK = TatumBscSDK({ apiKey: REPLACE_ME_WITH_TATUM_API_KEY })
```

Comment:
keep the API Key here - its default free one used for testing - its there for purpose